google-cloud-pubsub/support/doctest_helper.rb

```diff
@@ -562,10 +562,10 @@ def rec_message_json message, id = rand(1000000)
   }.to_json
 end
 
-def rec_messages_json message
+def rec_messages_json message, id = nil
```

Comment:
No, Pub/Sub is not using JSON. It is using GRPC and all data is received using the binary encoded protobuf objects.

These methods are here for convenience in instantiating the protobuf objects used in the documentation's code example tests.